qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -383,7 +383,7 @@ def _qubit_operator_to_terms(qubit_operator):
         *[
             (
                 coef,
-                qml.operation.Tensor(*[xyz2pauli[q[1]](wires=q[0]) for q in term])
+                qml.operation.Tensor(*[xyz2pauli[q[1]](wires=q[0]) for q in term])  # TODO: nonconsecutive wires
```

Comment:
When will this TODO be addressed? As I understand it, it will have buggy behaviour if the user does not have consecutive wires?

Reply:
Yes, I am waiting for feedback on how quick we can refactor qchem, and if that's more than a few lines I will raise a warning for now, forcing qchem users to use consecutive wires.

Reply:
Sounds like a good plan :+1:

Reply:
Will need to raise the warning, this could take a bit longer...